docs/howto/push-notifications.md

```diff
@@ -170,6 +170,74 @@ idiosyncratic capitalization shown.
+   certificate, so you should send it to him and ask him to do that. He'll
+   follow something like [these
+   instructions](https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/establishing_a_certificate-based_connection_to_apns)
+   to upload your CSR and obtain an APNs SSL (Sandbox) certificate (not Sandbox
+   & Production). Greg will send you the certificate. (Note: When you created the CSR, Keychain Access also created
+   a private key. The CSR itself is not the private key, so it doesn't require
+   extra care to send over the Internet. As of 2020-03-02, the APNs certificate
+   doc incorrectly conflates the CSR and the private key. The CSR file contains
+   only the public key and a signed hash of the request itself; this can be
+   confirmed by extracting its contents with OpenSSL:
+
+   `openssl asn1parse -i -in CertificateSigningRequest.certSigningRequest`
+   
+   `openssl req -in CertificateSigningRequest.certSigningRequest -text -noout`
+   
+   .)
+
+3. Keychain Access will recognize that the certificate corresponds to your
+   private key. Select them both and choose to export them to a single .p12
+   (Personal Information Exchange format) file.
+
+4. Move the .p12 file into your zulip.git clone, in /zproject. Then ssh into the
+   Vagrant container (`vagrant ssh`) and convert the .p12 file into the .pem
+   file, with
+
+   `openssl pkcs12 -in Certificates.p12 -out apns-sandbox.pem -nodes`
+
+   (replacing Certificates.p12 with whatever your filename is). If
+   openssl isn't installed, run `sudo apt install openssl`. Then delete the .p12
+   file; you won't need it anymore.
+
+5. The .pem file contains your private key, so be sure you don't push it to
+   GitHub! One way this could be done is with the .gitignore file, but
+   .gitignore is itself version-controlled, and other people probably don't have
+   a .pem file with the same name. The .git/info/exclude file lets you tell Git
+   what to ignore, but it isn't in version control, so other people won't be
+   confused by its contents. Add a line with "zproject/apns-sandbox.pem" (or
+   whatever the path is for you). Confirm that the .pem file isn't being tracked
+   by Git; it should not show up at all when you run `git status`.
+
+6. Add a line with `APNS_CERT_FILE = "zproject/apns-sandbox.pem"` to
+   zproject/dev_settings.py. This lets Python use the certificate to communicate
+   with the APNs sandbox server.
+
+Now, restart the server, and you should be receiving notifications on your iOS
+development build! Be sure mobile notification settings are on, using the web
+app's settings interface (this will be easiest if your branch contains
+[zulip/zulip@23ba2b6](https://github.com/zulip/zulip/commit/23ba2b63c5c10f43b02a2bb2c470cc6ff597d839)).
+If it's still not working, please contact someone on the mobile team before
+trying the next workaround, as that one will mean a worse development experience
+for you.
```

Comment:
This warning isn't necessary, because the next one is quite impossible to do without our involvement :slightly_smiling_face: -- it involves uploading to TestFlight.

Possibly it'd be good to add a mention of that fact at the top of that next section.

I think the key thing to say here is: if it's not working, please say so so we can debug.

Reply:
Also, re "contact someone on the team": what we generally want to steer people toward is asking in a stream (like `#mobile`).